### PR TITLE
Fix SCP wiki

### LINKED ISSUE #4225: [Presence | Bug] SCP Wiki
The SCP Wiki presence is facing a number of issues. These issues are as follows:
1. Failure to display any activity on the English site due to wrong URLs supported
2. Several errors displaying activity in international, secondary site, or foreign branch sites
Below is a detailed explanation for both errors.

--

**English Site Error:**
The SCP Wiki presence is missing support for the URL scp-wiki.wikidot.com, which is the URL that the English site uses now. All other URLs for the English site, including those listed to work with the presence, redirect to the unsupported URL.
Old site URLs:
www.scp-wiki.net
www.scpwiki.com
New site URL:
www.scp-wiki.wikidot.com

**Steps to Reproduce:**
1. Download presence
2. Attempt to go to the English SCP Wiki through Google search or direct links to the current or former English site URLs
3. Notice that the URL is redirected to scp-wiki.wikidot.com, not scp-wiki.net or scpwiki.com
4. Notice that, in Discord, the presence does not activate for the site

**Expected Behavior:**
Due to the current URL of the English SCP Wiki site not being supported by the presence, activity on the site will not be displayed in any visits to the site.

--

**International/Secondary/Foreign Branch Website Errors:**
The presence displays the activity on other supported URLs with a handful of errors. The errors I have seen are as follows:
- Failure to display any homepage or forum activity
- Only displays article activity (SCPs, announcements, GOI Formats, etc.)
- Chance of short delay in showing activity when presence is activated
- Chance of activity display suddenly ceasing for unknown reason
These bugs are mostly consistent, with an exception to the last two listed bugs.

The following sites are affected by these bugs:
scp-wiki-cn.wikidot.com (Chinese branch)
scp-cs.wikidot.com (Czech branch)
fondationscp.wikidot.com (French branch)
scp-wiki-de.wikidot.com (German branch)
scp-int.wikidot.com (International Translation Archive)
fondazionescp.wikidot.com (Italian branch)
ko.scp-wiki.net (Korean branch)
scp-jp.wikidot.com (Japanese branch)
scp-wiki.net.pl (Polish branch)
scp-pt-br.wikidot.com (Portuguese branch)
scpfoundation.net (Russian branch)
scp-es.com (Spanish branch)
scp-th.wikidot.com (Thai branch)
scpclassic.wikidot.com (Old Article Preservation)
scpexplained.wikidot.com (Article Explanation Site)

The following site is NOT affected by these bugs:
scp-sandbox-3.wikidot.com (Current Author Sandbox Site)

**Steps to Reproduce:**
1. Open up any site from the list of affected sites
2. Notice that, in Discord, the presence does not activate
3. Open up any article, announcement, or similarly-formatted page
4. Notice that, in Discord, the presence activates and displays activity on the page, sometimes after a couple seconds of delay
5. Open a forum post of any kind
6. Notice that, in Discord, the presence does not activate and therefore does not display activity on the page

**Expected Behavior:**
The sites listed as affected should be expected to show the errors explained above. Homepages will likely not display in the presence, and only articles or similarly-formatted pages will display. However, the possibility of the presence suddenly deactivating despite following reproduction steps is slim but existent. The exact chances of this are unknown.

--

**Environment**
- OS: Windows 10
- Browser: Chrome
- Version: 92.0.4515.131 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by MrBartusek**
Fixes: #4225

See https://github.com/PreMiD/Presences/issues/4225#issuecomment-932191017 for more info

image

### CODE REVIEW

File: websites/S/SCP Wiki/dist/metadata.json [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
   "author": {
     "name": "ry",
     "id": "421371986824921109"
   },
+  "contributors": [
+    {
+      "name": "MrBartusek",
+      "id": "212988300137463809"
+    }
+  ],
```
Comment (Slowlife01) [inline websites/S/SCP Wiki/dist/metadata.json:12]: You need 25% change

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Show screenshot of it working

File: websites/S/SCP Wiki/dist/metadata.json (hunk 1 above)
Comment (MrBartusek) [reply websites/S/SCP Wiki/dist/metadata.json:12]: That's a literary 18 line file

**Comment by Slowlife01** [comment]
That's old one?

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**